Engine/Commands/InvokeScriptAnalyzerCommand.cs

```diff
@@ -8,6 +8,7 @@
 using System.Collections.ObjectModel;
 using System.Diagnostics.CodeAnalysis;
 using System.Globalization;
+using System.IO;
```

Comment:
Nit: With the last commit, this using statement is not needed any more.

Reply:
`using System.IO;` *does* appear to be required for pre-existing unqualified references to the type `System.IO.InvalidDataException`.

Reply:
Not any more after the last commit that removed this type. I pushed a last commit to fix this and do some minor cleanup and make the format consistent to the solution and merge then after this.